platform/darwin/src/MGLBackgroundStyleLayer.mm

```diff
@@ -84,13 +85,33 @@ - (void)removeFromMapView:(MGLMapView *)mapView
 
 #pragma mark - Accessing the Paint Attributes
 
+- (NSArray *)transitionKeys
```

Comment:
Since this method doesn’t have any dependencies on other instance methods, you can make this a class method. But I’m unclear on this method’s purpose. It appears to be unused.